database/dbSchoolSuppliesForm.php

```diff
@@ -43,5 +43,39 @@ function isBackToSchoolFormComplete($childID) {
     return $complete;
 }
 
+
+function getSchoolSuppliesSubmissions() {
+    $conn = connect();
+    $query = "SELECT * FROM dbSchoolSuppliesForm JOIN dbChildren USING(id);";
```

Comment:
When I search only by school supplies form, I get the listing of children from dbSchoolSupplies (which is good) but the additional child info for each row (first_name, last_name, dob, gender, medical_notes, and notes) is info that is tied to a completely different child.

<img width="1762" alt="Screenshot 2024-12-03 at 6 59 38 PM" src="https://github.com/user-attachments/assets/b482e341-7253-4b39-bec0-e3c4bc442c25">


For example, in dbSchoolSuppliesForm on my machine, Mary MacLeod is id = 7, but her child_id from dbChildren is 10. Damien Palmer's dbChildren id is 7, the same as Mary's dbSchoolSuppliesForm id. So the results of the above query on line 71 seems to be linking Damien and Mary's info together. It's taking Mary's dbSchoolSupplies info but also attaching Damien's dbChildren info, since Mary's school supplies id and Damien's dbChildren ids are the same